lib/asciidoctor-diagram/blockdiag/extension.rb

```diff
@@ -106,7 +106,10 @@ def blockdiag(tool, parent, source, format)
         alt_cmd_name = "#{tool.downcase}3"
 
         CliGenerator.generate_stdin(which(parent, cmd_name, :alt_cmds => [alt_cmd_name]), format.to_s, source.to_s) do |tool_path, output_path|
-          [tool_path, '-o', Platform.native_path(output_path), "-T#{format.to_s}", '-']
+          args = [tool_path, '-a', -o', Platform.native_path(output_path), "-T#{format.to_s}"]
```

Comment:
There's an opening single quote missing here in front of the `-o` which causes syntax errors when running the code. Could you fix that?